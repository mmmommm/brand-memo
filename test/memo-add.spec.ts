import MemoAdd from '@/views/memo-add.vue'
import Vuetify from 'vuetify'

import { createLocalVue, shallowMount } from "@vue/test-utils"

describe('memo-add.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })
  const wrapper = shallowMount(MemoAdd, {
    localVue,
    vuetify,
  })
  it('snapshot test', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})