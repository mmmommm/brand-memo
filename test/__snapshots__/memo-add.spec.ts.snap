// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`memo-add.vue <MemoAdd /> 1`] = `
<v-form-stub
  lazyvalidation="true"
>
  <layout-stub>
    <v-flex-stub
      md6=""
      sm12=""
      tag="div"
      xs12=""
    >
      <v-card-stub
        flat="true"
        loaderheight="4"
        tag="div"
      >
        <v-layout-stub
          tag="div"
        >
          <v-flex-stub
            class="ml-4"
            tag="div"
          >
            <v-card-title-stub>
               日付 
            </v-card-title-stub>
             
            <basetext-stub
              value="2020/01/01"
            />
             
            <v-card-title-stub>
               コード 
            </v-card-title-stub>
             
            <basenumberinput-stub
              input="1111"
              placeholder="7203"
              value=""
            />
             
            <v-card-title-stub>
               銘柄名 
            </v-card-title-stub>
             
            <basetextinput-stub
              input="dummy"
              placeholder="トヨタ自動車"
              value=""
            />
             
            <v-card-title-stub>
               時価総額（百万） 
            </v-card-title-stub>
             
            <basenumberinput-stub
              input="1"
              placeholder="21495773"
              value=""
            />
             
            <v-card-title-stub>
               浮動株式数（株） 
            </v-card-title-stub>
             
            <basenumberinput-stub
              input="1"
              placeholder="100000000"
              value=""
            />
             
            <v-card-title-stub>
               テーマ 
            </v-card-title-stub>
             
            <basetextinput-stub
              input="dummy"
              placeholder="自動運転"
              value=""
            />
             
            <v-card-title-stub>
               株価（円）
            </v-card-title-stub>
             
            <basenumberinput-stub
              input="1"
              placeholder="6500"
              value=""
            />
             
            <v-card-title-stub>
               会社URL 
            </v-card-title-stub>
             
            <basetextinput-stub
              input="https://dummy.ac.jp"
              placeholder="https://company.co.jp"
              value=""
            />
          </v-flex-stub>
        </v-layout-stub>
      </v-card-stub>
    </v-flex-stub>
     
    <v-flex-stub
      md6=""
      sm12=""
      tag="div"
      xs12=""
    >
      <v-card-stub
        class="ml-8"
        flat="true"
        loaderheight="4"
        tag="div"
      >
        <v-card-title-stub>
           理由 
        </v-card-title-stub>
         
        <v-textarea-stub
          backgroundcolor=""
          clearicon="$clear"
          cols="60"
          errorcount="1"
          errormessages=""
          loaderheight="2"
          messages=""
          outlined="true"
          placeholder="この会社はどのような会社か、投資をするに値するか、業績等からみてどうか。"
          rowheight="24"
          rows="35"
          rules=""
          successmessages=""
          type="text"
          value="dummy"
        />
         
        <v-btn-stub
          activeclass=""
          class="ml-12 my-12 add"
          disabled="true"
          outlined="true"
          tag="button"
          type="submit"
          xlarge="true"
        >
          
          Addmemo ＜
        
        </v-btn-stub>
      </v-card-stub>
    </v-flex-stub>
  </layout-stub>
</v-form-stub>
`;
